templates/explore/organizations.tmpl

```diff
@@ -6,10 +6,16 @@
 
 		<div class="ui user list">
 			{{range .Users}}
+				{{if (or (eq .Visibility 1) (and ($.SignedUser) (or (eq .Visibility 2) (and (.IsUserPartOfOrg $.SignedUserID) (eq .Visibility 3)) ($.IsAdmin))))}}
```

Comment:
I don't think it us good idea to filter out user list in view... it should be done in select to get only needed data, otherwise paging will be wrong and could be that you get 10 pages with one or two orgs per page

Reply:
paging won't be wrong because SQL matches this case, so the results passed into template is already filtered. I will remove this conditional though, because it is not needed.

Reply:
Updated